bookbotproject/witapi.py

```diff
@@ -30,5 +30,11 @@ def auto_train_by_File(file_):
 	for sentence in sentences:
 		client.message(sentence)
 
```

Comment:
Can you please add a comment line here and explain what the function returns briefly? Just to make things more clear for future use. :)